lib/i18n_rails_helpers/i18n_controller_helpers.rb

```diff
@@ -0,0 +1,24 @@
+
+module I18nControllerHelpers
+  def initialize
+    super
+    self.class.delegate :t_attr, to: :helpers
+    self.class.delegate :t_model, to: :helpers
+    self.class.delegate :t_action, to: :helpers
+  end
```

Comment:
`initialize` doesn't work on modules. You can probably just move the `delegate` calls directly inside the `module` block:

```ruby
module I18nControllerHelpers
  delegate :t_attr, :t_model, :t_action, to: :helpers
```

Reply:
I can try again, but it didn't work for me. Thats why I did it that way after trying exactly this...

Reply:
@toupeira At least here, it does not work.

Reply:
@toupeira I don't remember what was different, but it used to not work, and now it does work.
However, never ask, if something starts to work, right? 😉

Reply:
Hmm strange, maybe you ran into some of Rails' sometimes-weird class caching behaviour :)